src/org/sosy_lab/java_smt/api/ArrayFormulaManager.java

```diff
@@ -61,6 +61,30 @@ <TI extends Formula, TE extends Formula> ArrayFormula<TI, TE> store(
   <TI extends Formula, TE extends Formula> ArrayFormula<TI, TE> makeArray(
       String pName, ArrayFormulaType<TI, TE> type);
 
+  /** Create a new array constant with uninitialized values. */
+  <TI extends Formula, TE extends Formula, FTI extends FormulaType<TI>, FTE extends FormulaType<TE>>
+      ArrayFormula<TI, TE> makeArray(FTI pIndexType, FTE pElementType);
+
+  /** Create a new array constant with uninitialized values. */
+  <TI extends Formula, TE extends Formula> ArrayFormula<TI, TE> makeArray(
```

Comment:
Do we really need two methods for doing the same thing? Just packing two types into one array-type could be done on user-side. We might want to reduce the API at this place and only provide one method.

Reply:
I took the other makeArray() methods as the basis for this API change (https://github.com/sosy-lab/java-smt/blob/7a9ec2262b0fe34b588515a43b2add10d1afe4c4/src/org/sosy_lab/java_smt/api/ArrayFormulaManager.java#L53-L62). However, I see no other point (besides adhering to the style of existing methods) in keeping both of these, so if you agree, I can remove the one taking two types instead of a single array type.